spec/lib/areacodes_spec.rb

```diff
@@ -52,6 +52,14 @@
     end
   end
 
+  describe '.area_codes_by_state_code' do
+    it 'returns area_codes' do
+      codes = described_class.area_codes_by_state_code(1)
+      expected = [205, 251, 256, 334, 938].sort
```

Comment:
Maybe consider mocking the data so that the behavior is tested and not the data.  I guess it's unlikely the data will change much, but when it does, it's possible it'll break the test(s).